network/peer-monitoring-service/client/src/metrics.rs

```diff
@@ -3,20 +3,84 @@
 
 use aptos_config::network_id::PeerNetworkId;
 use aptos_metrics_core::{
-    register_histogram_vec, register_int_counter_vec, register_int_gauge_vec, HistogramVec,
-    IntCounterVec, IntGaugeVec,
+    histogram_opts, register_histogram_vec, register_int_counter_vec, register_int_gauge_vec,
+    HistogramVec, IntCounterVec, IntGaugeVec,
 };
 use once_cell::sync::Lazy;
 
 /// The special label TOTAL_COUNT stores the sum of all values in the counter
 pub const TOTAL_COUNT_LABEL: &str = "TOTAL_COUNT";
 
+// Histogram buckets for tracking average ping latencies (secs)
```

Comment:
nit: I think it would look better interleaved so that the buckets are right next to the thing they go with

Reply:
Updated -- don't feel super strongly though 😄